model/exceptions.py

```diff
@@ -0,0 +1,7 @@
+class IllegalAbstractClassInitCall(Exception):
+    """Attempted to instantiate an abstract class"""
```

Comment:
I would prefer: "Exception raised when an instantiation of an abstract class is attempted"

Reply:
This is just Python's standard. Here are some examples of built-in exceptions and their docstrings:

- `TypeError`: _Inappropriate argument type._
- `KeyError`: _Mapping key not found._
- `NameError`: _Name not found globally._
- etc.
The rationale is that by having something extending Exception, it becomes obvious that it's an "Exception raised when..."
That being said, I can rename it to `AbstractClassInstantiatedError`?

Reply:
Understood.
Yes, good idea, I think that would make it clearer